### ISSUE TITLE
still issue on debian/sid

### ISSUE THREAD

**Issue by brummer10** [moddevices/mod-plugin-builder#52]

**Comment by brummer10** [comment]
There is still a issue remain by build the mod-plugin-builder on debian/sid. 
it's:
```
./python -E -S -m sysconfig --generate-posix-vars ;\
if test $? -ne 0 ; then \
	echo "generate-posix-vars failed" ; \
	rm -f ./pybuilddir.txt ; \
	exit 1 ; \
fi
Segmentation fault
generate-posix-vars failed
```

which could be solved by this patch:
https://github.com/python/cpython/commit/0b91f8a668201fc58fa732b8acc496caedfdbae0

regards
hermann

**Comment by falkTX** [comment]
Are you able to provide a pull request? that would speed up things a lot.

**Comment by brummer10** [comment]
Okay, pull request is pushed.

**Comment by falkTX** [comment]
Fixed in #53, thanks again
